src/net.h

```diff
@@ -361,13 +361,8 @@ class CConnman
     //! that peer during `net_processing.cpp:PushNodeVersion()`.
     ServiceFlags GetLocalServices() const;
 
-    //!set the max outbound target in bytes
-    void SetMaxOutboundTarget(uint64_t limit);
     uint64_t GetMaxOutboundTarget();
-
-    //!set the timeframe for the max outbound target
-    void SetMaxOutboundTimeframe(uint64_t timeframe);
-    uint64_t GetMaxOutboundTimeframe();
+    std::chrono::seconds GetMaxOutboundTimeframe();
```

Comment:
It's a good idea to use `std::chrono` internally in net, but I think you can leave these interface functions `GetMaxOutboundTimeframe()` and `GetMaxOutboundTimeLeftInCycle()` the same and continue to just send a count of seconds to the caller. The only place these are used is in rpc/net, where the count of seconds is returned directly to the user.

Reply:
> The only place these are used is in rpc/net, where the count of seconds is returned directly to the user.

`GetMaxOutboundTimeLeftInCycle` is called in `CConnman::OutboundTargetReached`.

Reply:
Ah, of course.